src/lib/_imports/components/c-card.css

```diff
@@ -308,3 +308,12 @@
 :--c-card--image-bottom > p:last-of-type {
     margin-bottom: unset; /* to avoid combining margin with image */
 }
+
+/* Image: Contact Card */
+:--c-card--image-top h4 {
+    margin-bottom: 0.375em;
+}
+:--c-card--image-top h4 ~ p {
+    margin-top: 0px;
+    margin-bottom: 10px;
+}
```

Comment:
I isolated the styles to only those `<p>` tags within a contact card (an image top card with an `<h4>`), because I don't think this style was intended to affect all cards.

Reply:
Looks good